spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/CastingRuleInterpreter.scala

```diff
@@ -59,7 +59,7 @@ case class CastingRuleInterpreter(rule: CastingConformanceRule) extends RuleInte
           c.cast(rule.outputDataType)
         }, c => {
           when(c.isNotNull.and(c.cast(rule.outputDataType).isNull),
-            callUDF(UDFNames.confCastErr, lit(rule.outputColumn), c.cast(StringType)))
+            call_udf(UDFNames.confCastErr, lit(rule.outputColumn), c.cast(StringType)))
```

Comment:
The `call_udf` is no camel case. Is here reason for that?

Reply:
Spark library function. Their decision 😉 . `callUDF` is deprecated.